README.rst

```diff
@@ -1,130 +1,165 @@
 
-Installation
-------------
+|PyPI| |Windows| |Linux|
 
-To install *coala* with the official set of analysis routines you can simply run
-``pip3 install coala-bears``.
+-----
 
-*coala* only, without the official bears, can be installed with
-``pip3 install coala``. If you need more information about the installation and
-dependencies, take a look at our `installation documentation
-<http://coala.rtfd.org/en/latest/Users/Install.html>`__.
+=====
+Usage
+=====
 
-The latest code from master is automatically deployed to PyPI as a
-development version. Get it with ``pip3 install coala --pre``.
+To get the list of arguments and their meaning, run:
 
-|PyPI|
+.. code-block:: bash
 
-Authors
--------
+    $ coala --help
 
-*coala* is maintained by a growing community. Please take a look at the
-meta information in `setup.py <setup.py>`__ for current maintainers.
+To run coala over your project, create a ``.coafile``, the project-specific
+configuration file that will store all your settings for coala. A sample
+``.coafile`` will look something like this:
 
-Project Status
---------------
+.. code-block:: bash
 
-|Linux Build Status| |Windows Build status|
+    [Default]
+    files = src/**/*.py
+    bears = SpaceConsistencyBear
+    use_spaces = True
 
-|Scrutinizer Code Quality| |codecov.io|
+Store this file in the project's root directory and run coala:
 
-|Documentation Status| |Gitmate|
+.. code-block:: bash
 
-Newcomers Guide and Getting Involved
-------------------------------------
+    $ coala
 
-If you are new and would like to contribute, read our `Getting Involved Site
-<http://coala.readthedocs.org/en/latest/Getting_Involved/README.html>`__!
+Please read our
+`coafile specification <http://coala.readthedocs.io/en/latest/Users/coafile.html>`__
+to learn more.
 
-We appreciate any help! Feel free to message us on
-`gitter <https://gitter.im/coala-analyzer/coala>`__. If you have any
-questions we're happy to help you!
+If you want to quickly test some settings without a ``.coafile``, you
+can run:
 
+.. code-block:: bash
 
-License
--------
+    $ coala --files=setup.py --bears=SpaceConsistencyBear -S use_spaces=True
 
-|AGPL|
+-----
+
+=======
+Support
+=======
+
+Feel free to contact us at our `Gitter channel <https://gitter.im/coala-analyzer/coala>`__, we'd be happy to help!
+
+You can also drop an email at our
+`mailing list <https://github.com/coala-analyzer/coala/wiki/Mailing-Lists>`__.
 
-This code falls under the GNU Affero General Public License as published
-by the Free Software Foundation, either version 3 of the License, or (at
-your option) any later version.
+|gitter|
 
-Please note that some files or content may be copied from other places.
-Most of them are GPL compatible. There is a small portion of code in the
-tests that falls under the Creative Commons license, see
-https://creativecommons.org/licenses/by-sa/3.0/deed.de for more
-information.
+-----
+
+================
+Getting Involved
```

Comment:
imo this section should be before ``Support``, that one is kind of what youd want to see at the end of the file. dont forget to swap in the toctree too, up there.